File: humbugg/Humbugg.Web/ClientApp/src/app/models/group.ts
import { GroupMember } from '../models/groupMember';
import { GroupRule } from "../models/groupRule";

export class Group {
  id: string;
  name: string;
  secretQuestion : string;
  secretQuestionAnswer: string;
  groupRules: Array<GroupRule>;
  groupMembers: Array<GroupMember>;
  signUpDeadline: string;
  signUpDeadlineDisplay: string;
  eventDate: string;
  eventDateDisplay: string;
  spendingLimit: number;
  description: string;
}

File: humbugg/Humbugg.Api/Models/Group.cs
using System;
using System.Collections.Generic;
using MongoDB.Bson.Serialization.Attributes;

namespace Humbugg.Api.Models
{
    public class Group : BaseModel
    {
        public string Name { get; set; }
        public string SecretQuestion { get; set; }
        public string SecretQuestionAnswer { get; set; }
        public List<GroupRule> GroupRules { get; set; }
        [BsonIgnore]
        public List<GroupMember> GroupMembers { get; set; }
        public DateTime SignUpDeadline { get; set; }
        public DateTime EventDate { get; set; }
        public decimal SpendingLimit { get; set; }
        public string Description { get; set; }
    }
}
